adapters/repos/db/index_integration_test.go

```diff
@@ -15,218 +15,238 @@
 package db
```

Comment:
Also not sure if possible, but can we set a node to readonly and do an acceptance test?

Reply:
Should be possible at an end2end level. Since we can explicitly mark shards (or all shards of a class) RO, the user journey (all through public APIs) could be:

1. Create a class
2. Mark all shards of a class RO
3. Delete class